code/game/objects/items/stunbaton.dm

```diff
@@ -90,7 +90,7 @@
 /obj/item/melee/baton/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
 	..()
 	//Only mob/living types have stun handling
-	if(turned_on && prob(throw_stun_chance) && iscarbon(hit_atom))
+	if(turned_on && prob(throw_stun_chance) && isliving(hit_atom) && !iscyborg(hit_atom))
 		baton_effect(hit_atom)
```

Comment:
What does this actually effect?

Reply:
batons don't just deal stamina damage and knock people down; they also cause them to stutter, confuse them, etc.

Reply:
also, spiders care about the stamina damage

Reply:
Why not cyborgs?

Reply:
cyborgs dont take stamina damage no? they only stun.

Reply:
> Why not cyborgs?

because the melee logic for stun batons excludes cyborgs but includes most other mobs

Reply:
But baton_effect has hardcoded behavior for them. What the fuck
![image](https://user-images.githubusercontent.com/58055496/103437091-dc3d7880-4bd7-11eb-9396-d59292d61647.png)

Reply:
silicons arent carbon, the old code didnt apply to silicons, im not sure if that was intended or not but silicons dont recieve the same effects carbon mobs would when stunned, hence why im assuming it was intended for that to not call when hitting a silicon, there is probably a different line somewhere thats supposed to apply the silicon effects. though im not sure what exactly they are, do stun batons actually stun borgs still? im not actually sure what effects silicons are even supposed to get when hit with stun batons anymore

Reply:
> But baton_effect has hardcoded behavior for them. What the fuck
> ![image](https://user-images.githubusercontent.com/58055496/103437091-dc3d7880-4bd7-11eb-9396-d59292d61647.png)

no, that's for if a borg is USING the baton, not for if it's being hit by one

Reply:
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh